### PR TITLE
Mbarba/simpler acc

### PR DESCRIPTION
**Pull request by MatBarba**
- Only keep one version of `download_genome_meta` by getting the accession from the `db` earlier (already available in the db meta).
- Use a meta value for all processes, with expected keys id (for display/uniqueness) and accession (INSDC/RefSeq)
- Generation of the meta values is clearer

### CODE REVIEW

File: pipelines/nextflow/modules/schema/check_json_schema_db.nf [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import java.io.File
+// import java.io.File
```
Comment (ens-LCampbell) [inline pipelines/nextflow/modules/schema/check_json_schema_db.nf:16]: Do we still need this line ?

**Review by ens-LCampbell — APPROVED** [review]
Overall I think a much cleaner implementation, thanks for unifying those datasets modules nice work ! 👍 

An aside: Do we need to integrate these changes into genome_prepare pipeline ?

File: pipelines/nextflow/modules/schema/check_json_schema_db.nf (hunk 1 above)
Comment (MatBarba) [reply pipelines/nextflow/modules/schema/check_json_schema_db.nf]: no indeed

**Review by JAlvarezJarreta — APPROVED** [review]
Couple of tiny details, but looks good otherwise. Like the improvements, definitely something worth pursuing in every pipeline/module.

**Pull request merged**